modules/metagetta/src/cljdoc_analyzer/metagetta/clojure.clj

```diff
@@ -145,9 +157,13 @@
       :skip-wiki    - legacy synonym for :no-doc"
 
   ([path] (read-namespaces path {}))
-  ([path {:keys [exception-handler]
+  ([path {:keys [exception-handler exclude-with]
            :or {exception-handler (partial utils/default-exception-handler "Clojure")}}]
    (let [path (utils/canonical-path path)]
      (->> (io/file path)
           (find-namespaces)
-          (mapcat #(read-ns % path exception-handler))))))
+          ;; shot at excluding namespaces before analysis/load
+          (remove-with-meta exclude-with)
+          (mapcat #(read-ns % path exception-handler))
```

Comment:
Since the tests are working I probably am just missing something but I'd have expected that we'd need to change `read-ns` here to support the fact that now `find-namespaces` returns maps. But I guess it always returned maps :)

Reply:
Hmm... thanks, will verify all is as intended.

Reply:
Oh, right... `find-namespaces` still returns namespaces, the difference is that they now have relevant metadata attached.